subsys/nrf_security/src/core/Kconfig

```diff
@@ -7,16 +7,23 @@
 choice PSA_CORE
 	prompt "PSA Core implementation"
 
+config PSA_CORE_DISABLED
+	bool
+	prompt "No PSA core (for SSF crypto client support)"
```

Comment:
```suggestion
	bool "No PSA core (for SSF crypto client support)"
```

Reply:
I did that for both this and the other config in this file which used prompt.